index.bs

```diff
@@ -813,6 +814,41 @@ enum AudioContextState {
 </table>
 </div>
 
+<pre class="idl">
+ enum AudioContextRenderSizeCategory {
+	"default",
+	"hardware"
+};
+</pre>
+
+<div class="enum-description">
+<table class="simple" dfn-for="AudioContextRenderSizeCategory" dfn-type="enum-value">
+	<thead>
+		<tr>
+			<th scope="col" colspan="2">
+				Enumeration description
+			</th>
+		</tr>
+	<tbody>
+		<tr>
+			<td>
+				"<dfn>default</dfn>"
+			<td>
+				The AudioContext's <a>render quantum size</a> is the default value of 128
+				frames.
+		<tr>
+			<td>
+				"<dfn>hardware</dfn>"
+			</td>
+			<td>
+				The User-Agent picks a <a>render quantum size</a> that is best for the
```

Comment:
I believe this value can be implementation specific. (possible privacy discussion)

Reply:
It's usually device specific -- but it's indeed something that can be used for fingerprinting. I'll add something.